src/data/text/abilities.h

```diff
@@ -508,20 +508,20 @@ const u8 gAbilityNames[ABILITIES_COUNT_GEN8][ABILITY_NAME_LENGTH + 1] =
     [ABILITY_ICE_FACE] = _("Ice Face"),
     [ABILITY_POWER_SPOT] = _("Power Spot"),
     [ABILITY_MIMICRY] = _("Mimicry"),
-    [ABILITY_SCREEN_CLEANER] = _("ScreenCleane"),
+    [ABILITY_SCREEN_CLEANER] = _("ScreenCleanr"),
     [ABILITY_STEELY_SPIRIT] = _("SteelySpirit"),
     [ABILITY_PERISH_BODY] = _("Perish Body"),
-    [ABILITY_WANDERING_SPIRIT] = _("WanderingSpi"),
+    [ABILITY_WANDERING_SPIRIT] = _("WandrngSprit"),
     [ABILITY_GORILLA_TACTICS] = _("GorillaTacti"),
-    [ABILITY_NEUTRALIZING_GAS] = _("Neutralizing"),
+    [ABILITY_NEUTRALIZING_GAS] = _("NutrlizngGas"),
```

Comment:
personally i like `NeutrlzngGas` but i'm not too attached to it (my personally preference would be `Neutral Gas` but that breaks MMBN convention)

Reply:
If I can choose, I prefer `NeutrlzngGas` too, honestly.